trytond/trytond/backend/sqlite/table.py

```diff
@@ -328,6 +328,22 @@ def set_indexes(self, indexes, concurrently=False):
                 cursor.execute('DROP INDEX %s' % _escape_identifier(name))
         self.__indexes = None
 
+    def dump_indexes(self, indexes, file, concurrently=False):
+        for index in indexes:
+            translator = self.index_translator_for(index)
+            if translator:
+                name, query, params = translator.definition(index)
+                name = '_'.join([self.table_name, name])
+                name = 'idx_' + self.convert_name(name, reserved=len('idx_'))
+                if not params:
+                    file.write(
+                        'CREATE INDEX IF NOT EXISTS %s ON %s %s' % (
```

Comment:
missing newline / `;`.
Also, index / table names should be enclosed in double quotes to handle special characters (dashes) in names.

Reply:
That's what `_escape_identifier` does